west.yml

```diff
@@ -0,0 +1,14 @@
+manifest:
+  self:
+    path: example-application
+
+  remotes:
+    - name: zephyr
```

Comment:
Confusing to use the same identifier as the name of the zephyr repository; maybe rename to zephyrproject-rtos?

Reply:
fixed